lib/matplotlib/backends/_backend_tk.py

```diff
@@ -609,6 +609,8 @@ class NavigationToolbar2Tk(NavigationToolbar2, tk.Frame):
     """
     def __init__(self, canvas, window):
         self.canvas = canvas
+        # Avoid using this (prefer self.canvas.manager.window), so that Tool
```

Comment:
Actually I think this attribute should be deprecated (because it is not consistent across backend implementations, so any cross-backend code will need to use `self.canvas.manager.window` anyways, and that doesn't seem to be so hard to type that you'd also want a property for it (if anything, making the NavigationToolbar APIs consistent across toolkits should be the object of another PR)).

But that's orthogonal to the PR, because the point of the comment is that sometimes, NavigationToolbar methods get called without a true NavigationToolbar as first argument, but a proxy created with _make_classic_style_pseudo_toolbar(), and these won't have a window attribute.